### DISCUSSION TITLE
[Announcement] Unofficial Community Discord

### DISCUSSION THREAD

**Discussion by mcmonkey4eva** [oobabooga/text-generation-webui#600] [Show and tell]
Ooba has expressed that he doesn't want to run an official Discord, but I think many in this community would appreciate having one, so I went ahead and created an Unofficial Community Discord for the text generation webui!

# Join @ https://discord.gg/u27RhJECcW

We have general chat channels, a help forum channel, development related channels, etc.
Whether you're a contributor, extension author, new user looking for help, or just an average user wanting to hang out with other people interested in the same text-gen topic as you, this Discord is the place to be, so come on in!

(Note: if ooba in the future ever changes his mind and wants to have an official Discord, I'd be happy to transfer this over)

**Comment by tensiondriven** [comment]
I agree with the desire to not run an official discord, and I get that there's probably value in it - My concern is that for projects like this, a lot of the value that we create as a community will be trapped in discord, where it would otherwise reside in more accessible resources like Github Issues, Github Discussions, etc.

I wonder if Discord's expectation of immediate responses creates an environment where people are less likely to research, contribute, and help each other; where people become conditioned to drop in, ask a q, and go on their way without creating any kind of value for others.  (Not trying to sh*t on Discord, just wanted to offer this perspective.)  I guess personally I've never gotten a lot of value out of Discord, and when I've tried, the process of getting information been a mess.  I don't want to discourage you though - just wanted offer some thoughts.

**Reply by mcmonkey4eva** [reply]
It's important to consider the scale of a community and who's in it - and the expectation that this is a community that's going to grow a lot in the coming days.
*Right now* it's fair to argue that it's valuable to keep things on GitHub, but as things grow more and more if we encourage people to use GitHub, they're going to more and more often use it for things that they should use *google* for. Check the Automatic1111 GitHub and see how much of that platform is usable - it's not, it's a mess ... thousands of nonsense issue posts, because there are so many inexperienced users flooding it with no better outlet. Unofficial communities have sprung up (r/StableDiffusion, civitai, etc.) with Discords and all that are able to catch and filter a big chunk of it (albeit not enough, would help to have more officialness to those side communities)

I've been involved in other communities that have official discords split off, and their githubs are *clean*. Any nonsense on there gets closed with "use the Discord" and its done with. Meaning you open the GitHub page and see important and valuable discussions, bugs, feature requests front & center.

The goal for the GitHub side of things should be a focus on actually important conversations. Issues should have real bug reports and feature requests, things like that - not the flood of basic user support that could be solved by sufficient google and/or critical thinking anyway.

The other value of Discord is providing a community outlet for the things like - basic conversations, initial ideas where somebody wants to talk something out but doesn't have it fully formed in their head... basic 'sense of community' things in general (on GitHub it's just a platform where we actually accomplish things, on Discord it's a _community_ ie just people with a shared interest hanging out).

The sense of community provided by Discord provides a source of growth for the project and interest in contributing it, that the GitHub on its own would not so much achieve.

Already in the first days of this Discord existing we've had over a hundred people join, many of them actively chatting about their personal interests and goals (what they want to *use* text gen webui for) and basic support topics (things like how to install basics). We've had people take an interest in contributing to the project on GitHub *thanks to* the Discord interactions we're having as a community.



-----



TLDR: GitHub posts and Discord groups are two different platforms with two different goals filling two different needs for two different sets of people. You cannot entirely replace one with the other, in either direction. When done well, their coexistence can be mutually beneficially.

**Comment by oivio** [comment]
I do agree with @tensiondriven 
Also bit out of topic. Recently I did hear that Discord it self do train own AI model on existing user data.

**Reply by tensiondriven** [reply]
Oh, fascinating.  Of course, we know that GitHub does this too, and GitHub has our source, not just our ideas.

A LLM trained on all discord conversations would be a pretty valuable thing.  Or, a pretty annoying thing :)

**Comment by oobabooga** [comment]
@mcmonkey4eva I am interested in joining the discord server to have a dev room. Could you make me admin?

This is my username (I'm not good with discord, is that enough?) `oobabooga#0748`

**Comment by ghost** [comment]
While having a Discord is a great way to get quick informal help or chat with other users, is anyone a bit concerned about people using it like a forum? I'm seeing members post guides/etc. there and problem with that is that the information is not searchable or accessible outside of Discord, unlike the publicly accessible and indexed Github discussions we're using now.

**Reply by oobabooga** [reply]
I also dislike keeping information not exposed to the web/search engines.

**Comment by FartyPants** [comment]
My beef with discord is that any useful info gets pushed back in a stream of so many messages posted a day. But it would be good place to share stuff.
